src/CollisionSolverIsaac.lib.s

```diff
@@ -0,0 +1,79 @@
+.DEFINE RECOVERYTIME 30
+
+; //// Collision Solver for Isaac \\\\
+collisionSolverIsaac :
+	push c
+	push d
+	push e

```

Comment:
same here

Reply:
In this function e is overwritten (line 42)